mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go

```diff
@@ -1587,6 +1587,27 @@ func TestAccSQLDatabaseInstance_sqlMysqlDataCacheConfig(t *testing.T) {
 	})
 }
 
+func TestAccSQLDatabaseInstance_sqlPostgresDataCacheConfig(t *testing.T) {
+	t.Parallel()
+	instanceName := "tf-test-enterprise-plus" + acctest.RandString(t, 10)
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccSqlDatabaseInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testGoogleSqlDatabaseInstance_sqlPostgresDataCacheConfig(instanceName),
```

Comment:
Let's add the check for `data_cache_enabled` property was set or not.

Reply:
Done